### PR TITLE
Header function

### PR DESCRIPTION
**Pull request by dylanlcole**
some footer fixes here as well

### CODE REVIEW

File: src/app/components/app.less [hunk 1]
```diff
@@ -22,7 +22,27 @@
...
+	text-align: left;
+}
+.right{
+	text-align: right;
+}
 
+.hide-mobile {
+	display: none;
+	flex-grow: 1;
+  padding: 0px 10px;
```
Comment (Riffmeister) [inline src/app/components/app.less]: Nitpick: Alignment. May just be github.

File: src/app/components/app.less [hunk 2]
```diff
@@ -22,7 +22,27 @@
...
+}
+.make-visible-mobile{
+	display: unset;
+	flex-grow: 1;
+  padding: 0px 10px;
```
Comment (Riffmeister) [inline src/app/components/app.less]: Nitpick: Same as above.

File: src/app/components/app.less [hunk 3]
```diff
@@ -117,25 +103,28 @@ li a:hover, li a:active {
 	display: flex;
 	justify-content: center;
 	background-color: #fff;
+	min-height: calc(100vh - 100px);
```
Comment (Riffmeister) [inline src/app/components/app.less:134]: What are you doing here?

Comment (dylanlcole) [reply src/app/components/app.less:134]: that does not show up in my code, probably a github thing

**Review by Riffmeister — APPROVED** [review]

**Pull request merged**